src/main/resources/alfresco/subsystems/Messaging/default/messaging-context.xml

```diff
@@ -68,6 +68,9 @@
     <!-- Any RouteBuilder in this package will be fired up -->
     <context:component-scan base-package="org.alfresco.messaging.camel.routes"/>
 
+    <!-- Any RouteBuilder in this enterprise package will be fired up -->
+    <context:component-scan base-package="org.alfresco.enterprise.repo.rendition2.routes"/>
```

Comment:
As pointed by @cristianturlica the name might be more generic like `org.alfresco.enterprise.repo.routes`.